dispatch/tests/test_settings.py

```diff
@@ -0,0 +1,43 @@
+SECRET_KEY = 'fake-key'
+
+INSTALLED_APPS = [
+    'tests',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'rest_framework',
+    'rest_framework.authtoken',
+    'dispatch',
+    'dispatch.apps.api',
+    'dispatch.apps.content',
+    'dispatch.apps.core',
+    'dispatch.apps.frontend'
+]
+
+BASE_URL = 'http://test.dispatch/'
+
+ROOT_URLCONF = 'dispatch.urls'
+DISPATCH_PROJECT_MODULE = None
+
+AUTH_USER_MODEL = 'core.User'
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': 'test_dispatch',
+    }
+}
+
+REST_FRAMEWORK = {
```

Comment:
it might make sense to add a comment here and in the actual settings reminding to keep them in sync -- given that some tests are really just testing that the correct permissions are being applied.